### ISSUE TITLE
Analyse risks of making AB  fields for schools optional

### ISSUE THREAD

**Issue by claire-hughez** [DFE-Digital/register-early-career-teachers#242]
## Why?
We ask schools for their appropriate body.

In ECF2, we're also considering asking them for an indication of who their lead provider and delivery partner are.

Historically, the appropriate body field has been optional. This is so schools could still come back and let us know the appropriate body later.

## What?
- Consider the risks and impacts of making AB, DP and LP optional when we collect it from schools
- Consider what options we have for how we treat these fields with schools - e.g. AB is optional but LP and DP are not
- Come up with any next steps for UR, data analysis or otherwise so we can derisk this decision
- Is there anything we can do to prevent the incorrect AB, DP or LP being entered?

## Helpful links or other information

On the optional fields of AB/LP/DP, one thing Mili suggested is considering if any validation could help to prevent schools from giving us incorrect info:
_Or there might be additional validation checks we can put in place but would need to be informed by some more analysis. For example, I think schools tend to work with DPs and ABs who are local to them (this needs checking though) so if they selected a DP/AB that was on the other side of the country, this might raise a flag or error message or something._

**Comment by gracecbrennan** [comment]
Emily is mid writing up a discussion to start working on this today - will require some team input on this.

**Comment by gracecbrennan** [comment]
@mason-emily to add link into ticket

**Comment by mason-emily** [comment]
https://github.com/DFE-Digital/ecf2/discussions/316

**Comment by gracecbrennan** [comment]
@claire-hughez to review discussion thread

**Comment by claire-hughez** [comment]
Split this into 2 tickets, one for ABs and one for LP/DP.

**Comment by claire-hughez** [comment]
Emily M to put time in to talk through it - maybe on Friday?

**Comment by gracecbrennan** [comment]
Waiting on Emma's analysis and will have a session next week on this.

**Comment by gracecbrennan** [comment]
Session put in for Thursday

**Comment by mason-emily** [comment]
We met to talk about whether the AB field should be mandatory or skippable in the journey for adding an ECT - see [here](https://github.com/DFE-Digital/register-early-career-teachers/discussions/316) for the options and risks/benefits/considerations.

We decided we want to start with the simplest option of making AB mandatory (option 1) and we will test how well that works along with the other changes currently being iterated to this part of the journey.

Next steps we're also planning to look at:
- Providing information upfront when a user wants to register an ECT on what details they'll need to complete the journey 

- Content/guidance to prevent schools from selecting an AB at random just to finish the journey (e.g. explaining why it needs to be accurate / impact if it's not) 

- Considering how we handle the data from the AB claiming an ECT vs the AB reporting from schools - we have an [analysis ticket](https://github.com/DFE-Digital/register-early-career-teachers/issues/348) already
